src/Client/CurlMultiHandler.php

```diff
@@ -16,6 +16,7 @@
  *
  * @property resource $_mh Internal use only. Lazy loaded multi-handle.
  */
+#[AllowDynamicProperties]
```

Comment:
Or declare the properties that are used?

Reply:
I did look at that, but the lazy loading `$_mh` is a world of pain that would need some significant refactoring :scream: